charts/spv-wallet-stack/charts/spv-wallet/templates/configmap.yml

```diff
@@ -8,91 +8,109 @@ data:
-        engine: {{  .Values.config.db.datastore.engine }} 
-        table_prefix: {{  .Values.config.db.datastore.table_prefix }} 
+        debug: {{ .Values.config.db.datastore.debug }}
+        engine: {{ .Values.config.db.datastore.engine }}
+        table_prefix: {{ .Values.config.db.datastore.table_prefix }}
       sql:
-        debug: {{  .Values.config.db.sql.debug }} 
-        driver: {{  .Values.config.db.sql.driver }} 
-        host: "{{  .Release.Name}}-postgresql-spv-wallet"
-        max_connection_idle_time: {{  .Values.config.db.sql.max_connection_idle_time }} 
-        max_connection_time: {{  .Values.config.db.sql.max_connection_time }} 
-        max_idle_connections: {{  .Values.config.db.sql.max_idle_connections }} 
-        max_open_connections: {{  .Values.config.db.sql.max_open_connections }} 
-        name: {{  .Values.postgresql.auth.database }}
-        password: {{  .Values.config.db.sql.password }} 
-        port: {{  .Values.config.db.sql.port }} 
-        replica: {{  .Values.config.db.sql.replica }} 
-        skip_initialize_with_version: {{  .Values.config.db.sql.skip_initialize_with_version }} 
-        table_prefix: {{  .Values.config.db.sql.table_prefix }} 
-        time_zone: {{  .Values.config.db.sql.time_zone }} 
-        tx_timeout: {{  .Values.config.db.sql.tx_timeout }} 
-        user: {{  .Values.postgresql.auth.username }}
-    debug: {{  .Values.config.debug }} 
-    debug_profiling: {{  .Values.config.debug_profiling }} 
-    disable_itc: {{  .Values.config.disable_itc }}
-    nodes:
-    {{- if .Values.config.nodes.deployment_id }}
-      deployment_id: {{ .Values.config.nodes.deployment_id }}
-    {{- else }}
-      deployment_id: 'spv-wallet-{{ printf "%s-$s"  (include "spv-wallet.fullname" .) .Values.global.domainName | adler32sum }}'
-    {{- end }}
+        debug: {{ .Values.config.db.sql.debug }}
+        driver: {{ .Values.config.db.sql.driver }}
+        host: "{{ .Release.Name }}-postgresql-spv-wallet"
+        max_connection_idle_time: {{ .Values.config.db.sql.max_connection_idle_time }}
+        max_connection_time: {{ .Values.config.db.sql.max_connection_time }}
+        max_idle_connections: {{ .Values.config.db.sql.max_idle_connections }}
+        max_open_connections: {{ .Values.config.db.sql.max_open_connections }}
+        name: {{ .Values.postgresql.auth.database }}
+        password: {{ .Values.config.db.sql.password }}
+        port: {{ .Values.config.db.sql.port }}
+        replica: {{ .Values.config.db.sql.replica }}
+        skip_initialize_with_version: {{ .Values.config.db.sql.skip_initialize_with_version }}
+        table_prefix: {{ .Values.config.db.sql.table_prefix }}
+        time_zone: {{ .Values.config.db.sql.time_zone }}
+        tx_timeout: {{ .Values.config.db.sql.tx_timeout }}
+        user: {{ .Values.postgresql.auth.username }}
+      sqlite:
+        database_path: {{ .Values.config.db.sqlite.database_path }}
```

Comment:
`<.Values.config.db.sqlite.database_path>: nil pointer evaluating interface {}.database_path`
Shouldn't we have a configuration for sqlite in values?

Reply:
i know i have fix for this